docs/management_api.md

```diff
@@ -59,7 +59,7 @@ curl -v -X POST "http://localhost:8081/models?initial_workers=1&synchronous=fals
 The synchronous call returns with HTTP code 200 after all workers have been adjusted.
 
 ```bash
-curl -v -X POST "http://localhost:8081/models?initial_workers=1&synchronous=true&url=https://<s3_path>/squeezenet_v1.1.mar"
+curl -v -X POST "http://localhost:8081/models?initial_workers=1&synchronous=true&url=https://<cloud_OR_noncloud-endpoint>/squeezenet_v1.1.mar"
```

Comment:
@gunandrose4u This will be more like <cloud_OR_noncloud_storage> and not endpoint, please change